File: Proyecto Escolar - Pagina Lenguajes de Programacion/tareas/java/Tarea_13.java
/* 
 * Alumno: Brandon Jahir Rojas Quintanilla
 * Matricula: 1635448
 * Hora: N2
 * Salón: 4201
 * Función: Busqeuda de numero en un vector
 */
 
import javax.swing.JOptionPane;
public class Tarea_13{
	public static void main(String[] arg){
		int i, size, num, contador, arreglo[];
		String entrada, vector;
		
		entrada = JOptionPane.showInputDialog("Ingrese tama\u00F1o del vector:");
		size = Integer.parseInt(entrada);
		
		arreglo = new int[size];
		vector = "[";
		
		for (i=0; i<size; i++){
			entrada = JOptionPane.showInputDialog("Ingrese un numero");
			arreglo[i]=Integer.parseInt(entrada);
			
			vector += arreglo[i] + ", ";
		}
		
		vector += "]";
		
		entrada = JOptionPane.showInputDialog("Ingrese numero a buscar:");
		num = Integer.parseInt(entrada);
		
		contador = 0;
		
		for (i=0; i<size; i++){
			if(arreglo[i] == num){
				contador++;
			}
		}
		
		JOptionPane.showMessageDialog(null, "Vector = " + vector +
		"\n\rBusqueda: " + num + "\n\rSu busqueda existe " + contador +" veces en el vector.");
	}
}

File: Proyecto Escolar - Pagina Lenguajes de Programacion/tareas/ruby/Tarea_13.rb
=begin
 * Alumno: Brandon Jahir Rojas Quintanilla
 * Matricula: 1635448
 * Hora: N2
 * Salón: 4201
 * Función: Busqueda en vector
=end

print("Ingrese el tamaño del vector ")
n=gets().to_i

array = Array.new(n)
mensaje="["
numero=""

for i in 1..n
  if numero == "" then
    print("Ingrese un numero ")
    numero=gets().to_i
    array[i]=numero
    mensaje+=numero.to_s+" "
  else
    print("Ingrese otro numero ")
    numero=gets().to_i
    array[i]=numero
    mensaje+=numero.to_s+" "
  end 
end 
mensaje+="]"

print("Ingresa el número que deseas buscar en el vector ")
buscar=gets().to_i

veces=0
for i in  (1..array.length)
  if array[i] == buscar then
    veces+=1
  end 
end 

print("El resultado del vector es de: "+mensaje.to_s+
"\nEl número que se busco fue el: "+buscar.to_s+
"\nLas veces que se repitio en el vector fue: "+veces.to_s)
